include/exadg/incompressible_navier_stokes/driver.h

```diff
@@ -72,7 +72,7 @@ get_dofs_per_element(std::string const & input_file,
                      unsigned int const  degree)
 {
   PressureDegree pressure_degree = PressureDegree::MixedOrder;
-  OperatorType   operator_type;
+  OperatorType   operator_type   = OperatorType::InverseMassOperator;
```

Comment:
This is a bugfix somewhat independent of the present PR. A few lines below (lines 91-92), an error is thrown if this variable is not initialized. @jh66637 FYI